dll/win32/comdlg32/lang/cdlg_Pt.rc

```diff
@@ -355,16 +355,16 @@ FONT 8, "MS Shell Dlg"
     GROUPBOX "Cópias", grp2, 160, 92, 120, 64, WS_GROUP
     LTEXT "Número de &cópias:", stc5, 168, 105, 68, 8
     ICON "", ico3, 210, 131, 76, 24, WS_GROUP
-    CONTROL "&Agrupar", chx2, "Button", BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP, 168, 118, 100, 12 //FIXME: accelerator collision &A
+    CONTROL "A&grupar", chx2, "Button", BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP, 168, 118, 100, 12
```

Comment:
Could be converted into `AUTOCHECKBOX ....` (see other existing examples above)
(And some other instances below.)